at-tanvo/Model/employee.rb

```diff
@@ -0,0 +1,25 @@
+require 'mysql2'
+require 'bcrypt'
+require 'pry'
+
+class Employee
+  attr_accessor :id, :name, :team, :role, :username, :password, :leave_balance
+  def initialize
+    @client = Mysql2::Client.new(host: 'localhost', username: 'root', password: '123456', database: 'TanVo_HumanResources')
+  end
+
+  def check_user(username)
+    sql = "SELECT * FROM `TanVo_HumanResources`.`employee` WHERE username = '#{username}'"
+    @client.query(sql)
+  end
+
+  def add(name, username, password, role)
+    sql = "INSERT INTO `TanVo_HumanResources`.`employee`(name, role, username, password, leave_balance) VALUES('#{name}','#{role}','#{username}','#{password}',10)"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/aaa8a77d0d40b4c4a78140482c3fa32e36501a75/at-tanvo/Model/employee.rb#L17"></span>Metrics/LineLength: Line is too long. [163/160] (https://github.com/rubocop-hq/ruby-style-guide#80-character-limits)</td>
    </tr>
  </tbody>
</table>